src/test/java/ibm/jceplus/junit/base/BaseTestDESede.java

```diff
@@ -78,7 +78,16 @@ public BaseTestDESede(String providerName, boolean supportsEncrypt) {
     //
     //
     public void setUp() throws Exception {
-        keyGen = KeyGenerator.getInstance("DESede", providerName);
+        try {
+            keyGen = KeyGenerator.getInstance("DESede", providerName);
+        } catch (NoSuchAlgorithmException nsae) {
+            if (providerName.equals("OpenJCEPlusFIPS")) {
+                assertEquals("no such algorithm: DESede for provider OpenJCEPlusFIPS", nsae.getMessage());
```

Comment:
How comes "no" is with a small n in this case? Is that a typo or is the exception message different?

Reply:
its a different exception message